.dockerignore

```diff
@@ -31,3 +31,6 @@ core*
 # git
 .git
 .gitignore
+
+# local dependencies
+dependencies/
```

Comment:
Where does this come from in the build process?

Reply:
`dependencies` directory is created by [setup.sh](https://github.com/The-OpenROAD-Project/OpenROAD-flow-scripts/blob/master/setup.sh#L20)